help.md

```diff
@@ -99,4 +98,5 @@ _This plugin does not contain any troubleshooting information._
 
 ## References
 
-* [Jenkins plugin page](https://plugins.jenkins.io/rapid7-insightvm-container-assessment)
\ No newline at end of file
+* [Jenkins plugin page](https://plugins.jenkins.io/rapid7-insightvm-container-assessment)
+* [Create a Rapid7 Platform API Key](https://insightappsec.help.rapid7.com/docs/get-started-with-the-insightappsec-api)
```

Comment:
Maybe we should use this more relevant link: https://insightvm.help.rapid7.com/docs/containers-cicd-plugin#section-generate-the-rapid7-api-key

or the general API key page: https://insight.help.rapid7.com/docs/managing-platform-api-keys

Reply:
Good call. Updated!